www/js/controllers/map-controller.js

```diff
@@ -26,41 +27,36 @@ angular.module('pvta.controllers').controller('MapController', function($scope,
       var loc = new google.maps.LatLng(location.lat, location.long);
       // Nested call: first, place the desired marker, then
       // add a listener for when it's tapped
-      addMapListener(placeDesiredMarker(loc), "here's what you're looking for!");
-      bounds.extend(loc);
+      addMapListener(placeDesiredMarker(loc, 'http://www.google.com/mapfiles/kml/paddle/go.png'), "here's what you're looking for!");
     });
 
   }
 
   // Almost the same as immediately above.
   // Query Cordova for current location first.
+  // Also fits the bounds to include everything we've added/removed from them
   var currentLocation = $cordovaGeolocation.getCurrentPosition(options).then(function(position){
     var myLocation = new google.maps.LatLng(position.coords.latitude, position.coords.longitude);
     bounds.extend(myLocation);
```

Comment:
My very last comment before merging: pursuant to your latest addition to lines 55-56, I think line 40 is now superfluous.